textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -31,81 +31,86 @@
 import edu.uci.ics.textdb.common.field.TextField;
 
 public class Utils {
-    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException{
+    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException {

```

Comment:
Are these changes due to reformatting?  If so, that's unfortunate since it makes the review harder.